programs/ssl/ssl_context_info.c

```diff
@@ -599,7 +603,12 @@ void print_deserialized_ssl_session( const uint8_t *ssl, uint32_t len,
                 ( (uint64_t) ssl[7] );
         ssl += 8;
         printf( "\tstart time     : " );
+#if defined(MBEDTLS_HAVE_TIME)
```

Comment:
(very minor, would understand if you don't want to do this) I notice that `print_time()` is only called the once, and has its argument cast from `uint64_t`. Another way round this would be to have `print_time()` take a `uint64_t` and then have the `#if` dance inside that function only - which would be simpler. There is already a default `printf( "unknown\n" );` in `print_time()` that could be kept outside the `#if`.

Reply:
Sure, that's a good improvement! I'll gladly add it, but as a separate PR, to not block this work that's already stalled.